schemas/input/csl-data.json

```diff
@@ -463,10 +465,27 @@
               "type": ["string", "number", "boolean"]
             },
             "literal": {
```

Comment:
I had wondered why we don't call this `name`, as we do in the CSL style schema?

Another decision you didn't make, but thought worth asking here.

Reply:
`name ` as opposed to `literal`? I think because the whole structure with `family`, `given`, `prefix`, etc. is a `name`. I had thought about whether we should rename this `organization` or `institution`.

Reply:
> Another decision you didn't make, but thought worth asking here.

Yeah, feel free to raise these. Now's a good time to review the data model.